### ISSUE TITLE
Focus support component identification

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#1190]
⚠️ This is only a sub task related to https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1189.

A list of all the components impacted needs to be created.

Results of this study should be mentioned in the main issue.
